charts/kube-prometheus-stack/values.yaml

```diff
@@ -2642,9 +2643,11 @@ prometheusOperator:
       strategy: {}
 
       # Ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
-      podDisruptionBudget: {}
-        # maxUnavailable: 1
-        # minAvailable: 1
+      podDisruptionBudget:
+        enabled: false
```

Comment:
theoretically, this is a breaking change. Before that, the object needs to ne non-empty to enabled PDB, now there is an enabled flag nessesary. Not sure about that, but I would recommend a major version bump here.